ui/src/features/project/pipelines/nodes/warehouse-node.tsx

```diff
@@ -111,15 +116,15 @@ export const WarehouseNode = (props: { warehouse: Warehouse }) => {
       <center>
         <Button
           size='small'
-          icon={<FontAwesomeIcon icon={faRefresh} />}
-          loading={warehouseState.refreshing}
+          icon={<FontAwesomeIcon icon={faRefresh} spin={warehouseState.refreshing} />}
           onClick={(e) => {
             e.stopPropagation();
             refreshWarehouseMutation.mutate({
               project: props.warehouse?.metadata?.namespace,
               name: props.warehouse?.metadata?.name
             });
           }}
+          disabled={warehouseState.refreshing}
```

Comment:
Why did you replace the loading prop with this?

Reply:
Before my last changes, I needed the icon to spin but not disabled, which is no longer requirement since I moved reconciling state outside button